core/src/executor/evaluate/mod.rs

```diff
@@ -294,6 +301,90 @@ async fn evaluate_function<'a, 'b: 'a, 'c: 'a, T: GStore>(
             let exprs = stream::iter(exprs).then(eval).try_collect().await?;
             f::concat(exprs)
         }
+        Function::Custom { name, exprs } => {
+            let custom_func = storage
+                .fetch_function(name)
+                .await
+                .map_err(|_| EvaluateError::UnsupportedFunction(name.to_string()))?;
+            if let Some(custom_func) = custom_func {
```

Comment:
```rust
let custom_func = storage
    .fetch_function(name)
    .await?
    .ok_or_else(|| EvaluateError::UnsupportedFunction(name.to_string()))?;
```
it would be enough to write like this. then we don't need to use `if let Some(custon_func) = custom_func` to wrap remaining codes.